conda-nighres.yml

```diff
@@ -1,36 +1,97 @@
-name: nighres
```

Comment:
This change is fine, but can we also have a "conda-nightres-develop.yml" that lists dependencies without versions so its easy to setup a dev environment to test (and upgrade) the versions eventually with new releases?

Reply:
Excellent idea, it would save me a few headaches...

Reply:
Will do.

Was also thinking that pinning dependencies like we are doing here may make it hard for nighres to "play nice" with another package that has other requirements. We might want to relax some of those with just a ">=".

This could also be an issue with different python version. Will at least test that wit python 3.5 - 3.10.

Reply:
Failing to create a proper conda env with python 3.10 because of packages conflicts.

https://github.com/Remi-Gau/nighres/runs/5798510905?check_suite_focus=true#step:3:1114